public/config/configuration.json

```diff
@@ -10,6 +10,8 @@
   "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
   "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
   "LEGAL_API_VERSION": "/api/v1",
+  "NAICS_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
+  "NAICS_API_VERSION": "/api/v2",
```

Comment:
This should work when deployed to Dev as I already updated your "vaults.json" file. LMK if it doesn't.

Reply:
copy that, thank you :)